tests/utils.ts

```diff
@@ -0,0 +1,315 @@
+    [Buffer.from("member"), mint.toBuffer()],
+    programId
+  );
+  return memberAccount;
+};
+
+export const getDistributionAccountPda = (
+  group: PublicKey,
+  paymentMint: PublicKey,
+  programId: PublicKey
+) => {
+  const [distributionAccount] = PublicKey.findProgramAddressSync(
+    [group.toBuffer(), paymentMint.toBuffer()],
+    programId
+  );
+  return distributionAccount;
+};
+
+export const getSaleAccountPda = (
+  group: PublicKey,
+  distribution: PublicKey,
+  programId: PublicKey
+) => {
+  const [saleAccount] = PublicKey.findProgramAddressSync(
+    [TEST_SALE, SALE, group.toBuffer(), distribution.toBuffer()],
+    programId
+  );
+  return saleAccount;
+};
+
+export const getListingAccountPda = (
+  seller: PublicKey,
+  mint: PublicKey,
+  programId: PublicKey
+) => {
+  const [listingAccount] = PublicKey.findProgramAddressSync(
+    [TEST_SALE, LISTING, seller.toBuffer(), mint.toBuffer()],
+    programId
+  );
+  return listingAccount;
+};
+
+export async function airdrop(
+  connection: Connection,
+  address: PublicKey,
+  airdropLamports: number,
+  commitment: Commitment = "confirmed"
+) {
+  const signature = await connection.requestAirdrop(address, airdropLamports);
+  const { blockhash, lastValidBlockHeight } = await connection.getLatestBlockhash(
+    commitment
+  );
+
+  await connection.confirmTransaction(
+    {
+      blockhash,
+      lastValidBlockHeight,
+      signature,
+    },
+    commitment
+  );
+}
+
+export async function getMinRentForWNSMint(
+  connection: Connection,
+  metaData: TokenMetadata,
+  type: string
+) {
+  // Size of MetadataExtension 2 bytes for type, 2 bytes for length
+  const metadataExtension = TYPE_SIZE + LENGTH_SIZE;
+  // Size of metadata
+  const metadataLen = pack(metaData).length;
+
+  // Size of Mint Account with extensions
+  const mintLen = getMintLen(
+    [
+      ExtensionType.MintCloseAuthority,
+      ExtensionType.MetadataPointer,
+      ExtensionType.TransferHook,
+      ExtensionType.PermanentDelegate,
+    ].concat(
+      type === "member"
+        ? [ExtensionType.GroupMemberPointer]
+        : [ExtensionType.GroupPointer]
+    )
+  );
+
+  // Minimum lamports required for Mint Account
+  return connection.getMinimumBalanceForRentExemption(
+    mintLen + metadataExtension + metadataLen
+  );
+}
+
+export async function createMint2022Ix(
+  connection: Connection,
+  mint: PublicKey,
+  authority: PublicKey,
+  payer: PublicKey
+) {
+  // Size of MetadataExtension 2 bytes for type, 2 bytes for length
+  const metadataExtension = TYPE_SIZE + LENGTH_SIZE;
+
+  const metadata: TokenMetadata = {
+    mint,
+    name: "USD Coin",
+    symbol: "USDC",
+    uri: "https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f736f6c616e612d6c6162732f746f6b656e2d6c6973742f6d61696e2f6173736574732f6d61696e6e65742f45506a465764643541756671535371654d32714e31787a7962617043384734774547476b5a777954447431762f6c6f676f2e706e67",
```

Comment:
can you use faker?

Reply:
Was trying to mimic USDC here hehe. But yeah for sure